### ISSUE TITLE
Mediaset infinity 

### ISSUE THREAD

**Issue by Seby68** [richardfrost/AdvancedProfanityFilter#431]
```json
{
  "mediasetinfinity.mediaset.it": [
    {
      "ignoreMutations": true,
      "mode": "elementChild",
      "parentSelector": "div#playerContainer > div > div > div > div > div",
      "simpleUnmute": true,
      "tagName": "#text",
      "videoSelector": "div#playerContainer video"
    },
    {
      "ignoreMutations": true,
      "mode": "elementChild",
      "parentSelector": "div#playerContainer> div > div > div > div > div > div > div > div > div>div",
      "simpleUnmute": true,
      "tagName": "DIV",
      "videoSelector": "div#playerContainer video"
    }
  ]
}
```

**Comment by Seby68** [comment]
This is the code . Thank you.

**Comment by richardfrost** [comment]
This is included in [Version 2.37.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.37.0). If anyone has a chance to test this out, feel free to reply here to report back how it works for you. Thanks!
